### ISSUE TITLE
SC1133: wrong error reported

### ISSUE THREAD

**Issue by laurent-simon** [koalaman/shellcheck#2281]
- Rule :  SC1133
- Version : 0.7.1
- [x] The [SC1133 rule wiki page](https://shellcheck.net/wiki/SC1133) does not already cover this.
- [x] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### Test code

```sh
#!/bin/bash

echo 'Ok' 
&>/dev/null echo 'Ko'
```
The two statements are distinct (no line breaking).

#### Unexpected error

```txt
$ shellcheck myscript
 
Line 4:
&>/dev/null echo 'Ko'
^-- SC1133: Unexpected start of line. If breaking lines, |/||/&& should be at the end of the previous one.
```

Apparently, there is a confusion with the `&` at the start of the second command. It's a redirection off all outputs (silent command), but shellcheck consider it as a line breaking case.

SC1133 must not report errors on valid code. What the Bash man page says on redirections (extract):

> redirection operators may precede or appear anywhere within a simple command or may follow a command.

So, when a `&` is immediately followed by a `>`, it should not be considered by the SC1133 rule.

In a general way,  a single `&`can never be a line breaking case. The `&` character should be considered for line breaking only if it is doubled.

**Comment by koalaman** [comment]
ShellCheck was checking if a command started with `&` or `|` because starting a command with `&`, `&&`, `|`, or `||` generally indicates a bug. Like you point out, this does not apply to Bash/Ksh `&>`, and this is now accounted for. Nice, thanks!

(ShellCheck <=0.7.2 still parsed this correctly, so it can be ignored with a directive with no ill effects.)
